indico/modules/designer/client/js/index.js

```diff
@@ -113,7 +115,13 @@ import {$T} from 'indico/utils/i18n';
           zIndex: this.zIndex,
         })
         .attr('data-type', this.type)
-        .text(this.type === 'fixed' ? this.text : itemTitles[this.type]);
+        .text(
+          this.type === 'fixed'
+            ? this.text
+            : this.type === 'fixed_image' && item.image_id
+            ? ''
+            : itemTitles[this.type]
+        );
```

Comment:
I think we can simplify this now that we are using placeholders.

```suggestion
        .text(itemTitles[this.type]);
```

Reply:
I kept this condition because the fixed text can take a user input, making the inputted text different from the item title, also when an image is uploaded for the fixed_image, it should only display the image and not still show the title. So the title should only be used when there is no image.

Reply:
Right, never mind, for some reason I assumed this was referring to the selected text.